draft-ietf-httpbis-digest-headers.md

```diff
@@ -1028,7 +1001,7 @@ Digest: sha-256=4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=
 ~~~
 {: title="Empty response with Digest"}
 
-## Client and Server Provide Full Representation Data, Client Uses id-sha-256.
```

Comment:
Is the comment about the second digest value at https://github.com/httpwg/http-extensions/pull/1779/files#diff-de34152d297a9eb29b85b24962949ba2e309478a5c32e1d94fd4c3ee27b4584dL1037 still correct with this change?